### PR TITLE
Add error log to upload service

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/9430

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-runtime/src/components/UploadInfo.vue [hunk 1]
```diff
@@ -141,6 +145,9 @@
         </li>
       </ul>
     </div>
+    <div v-if="infoExpanded && uploadErrorLogContent" class="upload-info-error-log oc-px-m oc-pb-m">
```
Comment (JanAckermann) [inline packages/web-runtime/src/components/UploadInfo.vue:148]: I don't think wrapping div is required

File: packages/web-runtime/src/components/UploadInfo.vue [hunk 2]
```diff
@@ -657,6 +677,10 @@ export default defineComponent({
     overflow-y: auto;
   }
 
+  .upload-info-items.has-errors {
```
Comment (JanAckermann) [inline packages/web-runtime/src/components/UploadInfo.vue:680]: Keep height consistent

**Review by JammingBen — APPROVED** [review]
The `margin-top` of the error box could be a bit less IMO, but not a deal breaker for me. Nice!

**Comment by JanAckermann** [comment]
> The `margin-top` of the error box could be a bit less IMO, but not a deal breaker for me. Nice!

Yes, that's the same for notifications (l). I don't have any issues changing that to m

**Pull request merged**